src/queries/models/rewards/createUserRewardsDetailsModel.ts

```diff
@@ -0,0 +1,3 @@
+import { createUserRewardsDetailsResponse } from '../../types/rewards';
+
+export const createUserRewardsDetailModel = (response: any): any => response;
```

Comment:
```suggestion
export const createUserRewardsDetailsModel = (response: UserRewardsDetailsResponse): UserRewardsDetailsResponse => response;
```

@abhishek-01k need type here

Reply:
@abhishek-01k resolved this

Reply:
okay